### PR TITLE
Update support for 128 numbers in untagged enums

### PR DESCRIPTION
**Pull request by Licenser**
This adds some support for untagged enums as mentioned in #1717  . But it causes some problems w/ serde-json and I'm not sure how to best handle this. Any suggestions?

### CODE REVIEW

File: serde/src/de/impls.rs [hunk 1]
```diff
@@ -141,12 +192,14 @@ macro_rules! impl_deserialize_num {
         visit_integer_method!(i32, visit_i32, from_i32, Signed, i64);
         visit_integer_method!(i64, visit_i64, from_i64, Signed, i64);
 
-        visit_integer_method!(u8, visit_u8, from_u8, Unsigned, u64);
+        visit_integer_method!(u8, visit_u8, from_u8, Unsigned, u64) ;
```
Comment (teor2345) [inline serde/src/de/impls.rs:195]: Nitpick: looks like there's an accidental extra space here

```suggestion
        visit_integer_method!(u8, visit_u8, from_u8, Unsigned, u64);
```

Comment (teor2345) [reply serde/src/de/impls.rs:195]: (But I guess rustfmt will get rid of all the extra spaces, so it might not really matter.)

Comment (Licenser) [reply serde/src/de/impls.rs]: thank you!

**Review by dtolnay — COMMENTED** [review]
I'll close this PR since it does not resolve the serde_json use case, as you noticed.

There have been some other PRs along the same lines which I've also closed -- please see #1679 and #1979. In particular https://github.com/serde-rs/serde/pull/1979#pullrequestreview-735514847 has a brief proposal of what direction to go with supporting this use case.

**Pull request closed without merging**